app/models/ability.rb

```diff
@@ -229,7 +227,7 @@ def initialize(user)
 
       # Extend default permission to see visible schools with public data to also add permission to
       # view visible schools in the same group that have data sharing set to be 'within_group'
-      can [:show, :show_pupils_dash, :show_management_dash], School, { data_sharing: :within_group, school_group_id: user.school.school_group_id, visible: true }
```

Comment:
Assume :show_management_dash was intentionally removed here?

Reply:
Yes, this setup apparently gave staff and volunteers access to some features on other schools that school admins didn't have which seemed odd. I could get it to trigger though, and have tests around the expected behaviour so I think we're OK.